AirCasting/CoreData/MeasurementStreamStorage.swift

```diff
@@ -121,6 +120,20 @@ final class HiddenCoreDataMeasurementStreamStorage: MeasurementStreamStorageCont
         return stream?.localID
     }
     
+    func getExistingSessionWith(_ sessionUUID: SessionUUID) throws -> SessionEntity? {
```

Comment:
I'm thinking wherher `With` is needed in this name. Maybe it could be `getExistingSession` or `getExistingSessionWithUUID`?